src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -99,6 +112,24 @@ async def do_delete(self, id):
         Delete Tunable of `id`.
         """
         tunable = await self._get_instance(id)
+        await self.lower(tunable)
+        if tunable['type'].lower() == 'sysctl':
```

Comment:
Dont think none of this is required as everything is handled in sysctl etc file?